lib/src/util/cache_package.dart

```diff
@@ -0,0 +1,26 @@
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:path/path.dart' as path;
+
+class CachePackage {

```

Comment:
Can I just know why this  .dart_tools cache is needed? Cause during version management of the app/code they will be ignored in git. Soo just curious to know why!!!

Reply:
That specific file contains info as to where all the dependencies are locally stored on the user's machine.
So by parsing the .dart_tools/package_config.json, I can find where at_app_flutter is stored.
You don't want to commit it because it is different for every developer.